machine/scripts/db.d/10-mysqld.sh

```diff
@@ -28,6 +28,7 @@ yum install -y mysql-server
 service mysqld start >> $BOOTSTRAP_LOG 2>&1
 mysql -uroot -e "
     GRANT ALL PRIVILEGES ON *.* TO 'root'@'localhost' WITH GRANT OPTION;
+    GRANT ALL PRIVILEGES ON *.* TO 'root'@'127.0.0.1' WITH GRANT OPTION;
```

Comment:
The root mysql user should never have access from an external machine, so yes, this line needs to be removed. I've verified the change in the cnf doesn't open the port externally (as expected) via `netstat -plut`